index.html

```diff
@@ -19,11 +21,26 @@
  <!-- navigation code here -->
 
   <div class="container home">
+    <div class="space-top">
+    <h1 class="top">SpaceWalkers</h1>
+    <nav class="my-mixin">
+      <a>Home</a>
+      <a>About Us</a>
+      <a>Blog</a>
+      <a>Contact</a>
+      <a>Sign in</a>
+    </nav>
```

Comment:
remember to add in the hrefs when you use <a> tags, even if its the # as a place holder